### PR TITLE
Fix #105

### LINKED ISSUE #105: Escape Hatch Login stopped working after Jenkins Upgrade to 2.266 (Spring Security Update JEP-227)
Recently upgraded for Jenkins 2.256 to 2.275, and updated all plugins at point of upgrade.  Had put the upgrade off for a little while after spectating issues relating to the [table to div layout change](https://issues.jenkins.io/browse/JENKINS-64072).  The OpenIdConnect plugin is currently at version 1.8.

After the upgrade, the escape hatch login provides a 401 when attempting login.  Setting a new password via Configure Global Security doesn't help.

No exceptions were logged on failed login.

### PR DESCRIPTION
**Pull request by Skiepp**
Fix #105

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/oic/OicUserProperty.java [hunk 1]
```diff
@@ -19,22 +20,22 @@ public static class Descriptor extends UserPropertyDescriptor {
...
 		public UserProperty newInstance(User user) {
 			LOGGER.fine("OicUserPropertyDescriptor.newInstance called, user:" + user);
-			return new OicUserProperty(user.getId(), new GrantedAuthority[0]);
+			return new OicUserProperty(user.getId(), new ArrayList<GrantedAuthority>());
 		}
 
 		@Override
 		public String getDisplayName() {
-			return Messages.openid_connect_user_property();
+			return "OpenID Connect user property";
```
Comment (halkeye) [inline src/main/java/org/jenkinsci/plugins/oic/OicUserProperty.java:28]: FYI, this is just removing translations, seems like an odd choice unless you are adding new strings.

File: src/test/java/org/jenkinsci/plugins/oic/ConfigurationAsCodeTest.java [hunk 2]
```diff
@@ -1,70 +0,0 @@
-package org.jenkinsci.plugins.oic;
```
Comment (halkeye) [inline src/test/java/org/jenkinsci/plugins/oic/ConfigurationAsCodeTest.java:1]: I would recommend keeping casc tests working, so you don't break people who use the plugin with config as code.

File: pom.xml [hunk 3]
```diff
@@ -3,17 +3,17 @@
...
-    <jenkins.version>2.60.3</jenkins.version>
-    <configuration-as-code.version>1.27</configuration-as-code.version>
-    <java.level>8</java.level>
+    <jenkins.version>2.346.1</jenkins.version>
+    <spotbugs.failOnError>false</spotbugs.failOnError>
   </properties>
 
   <artifactId>oic-auth</artifactId>
-  <version>1.9-SNAPSHOT</version>
+  <version>2.0.0</version>
```
Comment (halkeye) [inline pom.xml:16]: I think this will break, version should be SNAPSHOT until you do a release, which will update the version number

**Comment by Skiepp** [comment]
Thank you very very much @halkeye, I pushed all your change proposal

**Pull request merged**